knowledge-base/Azure/static-web-apps-deploy/action-security.yml

```diff
@@ -6,3 +6,4 @@ github-token:
   permissions:
     pull-requests: write
     pull-requests-reason: to comment on PRs # CHECKOUT: https://github.com/Azure/static-web-apps-deploy/blob/v1/action.yml#L32
+    pull-requests-if: ${{ contains(with, 'repo_token') }}
```

Comment:
We should not need this. The current logic already checks for this if the input is for github_token.